bson/testdata/fuzz/FuzzDecode/002ae7d43f636100116fede772a03d07726ed75c3c3b83da865fe9b718adf8ae

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
+[]byte("\x10\x00\x00\x00\v\x00\x00\x00\b\x00\x00\v\x00\x00\x00\x00")
```

Comment:
Are these `testdata/fuzz/FuzzDecode` files separate from the `bson-corpus` seeds? How are these added into the seed corpus?

Reply:
This data is different from testdata/bson-corpus. Three of these were interesting cases generated by running the fuzzer. One is BSON that encapsulates all types for an initial maximum code coverage in the style of the [encoding/json fuzz test](https://cs.opensource.google/go/go/+/master:src/encoding/json/fuzz_test.go).

From the [documentation](https://go.dev/security/fuzz/):
> seed corpus: A user-provided corpus for a fuzz test which can be used to guide the fuzzing engine. It is composed of the corpus entries provided by f.Add calls within the fuzz test, and the files in the testdata/fuzz/{FuzzTestName} directory within the package. These entries are run by default with go test, whether fuzzing or not.

Reply:
Great, thanks for the explanation.